pandas/tests/io/json/test_pandas.py

```diff
@@ -1315,27 +1190,32 @@ def test_sparse(self):
         expected = s.to_json()
         assert expected == ss.to_json()
 
-    def test_tz_is_utc(self):
+    @pytest.mark.parametrize(
+        "ts",
+        [
+            Timestamp("2013-01-10 05:00:00Z"),
+            Timestamp("2013-01-10 00:00:00", tz="US/Eastern"),
+            Timestamp("2013-01-10 00:00:00-0500"),
+        ],
+    )
+    def test_tz_is_utc(self, ts):
         from pandas.io.json import dumps
 
         exp = '"2013-01-10T05:00:00.000Z"'
 
-        ts = Timestamp("2013-01-10 05:00:00Z")
-        assert dumps(ts, iso_dates=True) == exp
-        dt = ts.to_pydatetime()
-        assert dumps(dt, iso_dates=True) == exp
-
-        ts = Timestamp("2013-01-10 00:00:00", tz="US/Eastern")
-        assert dumps(ts, iso_dates=True) == exp
-        dt = ts.to_pydatetime()
-        assert dumps(dt, iso_dates=True) == exp
-
-        ts = Timestamp("2013-01-10 00:00:00-0500")
         assert dumps(ts, iso_dates=True) == exp
         dt = ts.to_pydatetime()
         assert dumps(dt, iso_dates=True) == exp
```

Comment:
this is dependent on the original `ts` having no nanos?

Reply:
Hmm can you clarify? Not sure I understand your question

Reply:
all of the `Timestamp` objects I see have `ts.nanosecond == 0`, which means `ts.to_pydatetime()` is non-lossy.  The question is if we need to (test|worry about) cases with nonzero nanoseconds